inc/Auth/SyncPermissions.php

```diff
@@ -110,7 +112,7 @@ private static function check_post_sync_permissions(
 		// Check sync_post capability (will be mapped to edit_post via map_meta_cap)
 		/** @var bool|WP_Error $can_sync_post */
 		$can_sync_post = true;
-		if ( ! current_user_can( 'sync_post', $parsed_post_id ) ) {
+		if ( ! current_user_can( self::CAP_NAME, $parsed_post_id ) ) {
 			$can_sync_post = new WP_Error(
 				'insufficient_sync_permissions',
 				__( 'You do not have permission to sync this content', 'vip-real-time-collaboration' )
```

Comment:
should this be better described now the cap has changed from `sync_post`?

"You do not have permission to edit with other users"?

Reply:
Good opportunity for

"I'm afraid I can't let you do that ____"